packages/valory/skills/oracle_deployment_abci/payloads.py

```diff
@@ -99,13 +99,13 @@ def data(self) -> Dict:
         return dict(keeper=self.keeper)
 
 
-class ValidateOraclePayload(BaseTxPayload):
-    """Represent a transaction payload of type 'validate'."""
+class VotingOraclePayload(BaseTxPayload):
```

Comment:
why the change?

Reply:
I made the payload re-usable on a52dcf75115450df72b04421747205257d236d6c so that it can be re-used by the new round introduced on 40466c228ea308d895a06c20e6ff13d28c3564e0.

A `VotingPayload` should be implemented on the core at some point since it is always the same.